### PR TITLE
Add functions for handling addresses and more prometheus metrics for the Relay

### PR DESCRIPTION
**Pull request by kkirkov**

### CODE REVIEW

**Review by EmilIvanichkovv — CHANGES_REQUESTED** [review]
New utils from `feat(relay): Add new functions for handling addresses (3d581cc118c964531c353589c3f3e6137ad97f10) ` can live in dedicated module `libs/typescript/ts-utils/evm`

File: relay/implementations/publish_evm_transaction.ts [hunk 1]
```diff
@@ -72,32 +73,16 @@ export async function publishTransaction(
...
-          }
-        }
+        logger.info(
+          'Transaction uploaded at: ',
+          explorerUrls[chainName].tx(transaction.hash),
+        );
+        logger.info(
+          'Contract address: ',
+          explorerUrls[chainName].address(contract.address),
+        );
```
Comment (EmilIvanichkovv) [inline relay/implementations/publish_evm_transaction.ts:85]: This is smooth.

File: relay/utils/discord_monitor.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ async function checkContractUpdate(
     }
 
     if (delayInMinutes >= alertThresholdMinutes) {
-      const message = `@DendrETH Support Relay ⚠️ Alert: Contract on **${network}** hasn't been updated in ${delayInMinutes.toFixed(
+      const message = `<@&1285217827560620113> ⚠️ Alert: Contract on **${network}** hasn't been updated in ${delayInMinutes.toFixed(
```
Comment (EmilIvanichkovv) [inline relay/utils/discord_monitor.ts:59]: This is how you tag a discord role? This is the ID of the role I guess?

**Review by EmilIvanichkovv — COMMENTED** [review]
I guess more prometheus metrics = more good. Just one comment left

File: libs/typescript/ts-utils/common-utils.ts [hunk 3]
```diff
@@ -361,3 +361,74 @@ export function extractHostnameAndPort(url: string) {
...
+export type EthereumAddress = string & { __brand: 'EthereumAddress' };
+export const ethereumAddress = (
+  hexDataString: string,
+): EthereumAddress | null => {
+  const isValid = /^0x([0-9a-fA-F]{40})$/.test(hexDataString);
+  if (isValid) {
+    return hexDataString as EthereumAddress;
+  }
+  return null;
+};
```
Comment (EmilIvanichkovv) [inline libs/typescript/ts-utils/common-utils.ts:389]: Would be nicer to use effect/schema, but I think it is not used yet at DendrETH

**Comment by kkirkov** [comment]
> New utils from `feat(relay): Add new functions for handling addresses (3d581cc118c964531c353589c3f3e6137ad97f10) ` can live in dedicated module `libs/typescript/ts-utils/evm`

Thanks for the idea - it is done

**Review by Dimo99 — APPROVED** [review]

**Review by EmilIvanichkovv — APPROVED** [review]

**Pull request merged**